client/src/components/Navbar/index.js

```diff
@@ -35,15 +35,15 @@ const Navbar = () => {
 					{currentUser && location.pathname !== "/" && currentUser && location.pathname !== "/regions" ? (
 						<Navigation
 							history={history}
-							title={"Go back to regions"}
+							title={"Back to regions"}
 						/>
 					) : null}
 				</div>
 			) : <div className='navigation-btn-container'>
 				{currentUser && location.pathname !== "/" && currentUser && location.pathname !== "/regions" ? (
```

Comment:
{currentUser && location.pathname !== "/" && currentUser && location.pathname !== "/regions" ? 
Not sure? Do we need the second currentUser here?

Reply:
Hi Tayfun, good spot. thank you. I did remove it, it was repeating not necessary. Will push it now